api/controllers/ReferenceController.js

```diff
@@ -249,13 +246,12 @@ module.exports = {
     });
   },
 
-  getFlairs: function (req, res) {
-    Flair.find().exec(function (err, flairs) {
-      if (err) {
-        return res.serverError(err);
-      }
-      return res.ok(flairs);
-    });
+  getFlairs: async function (req, res) {
+    try {
+      return res.ok(await Flairs.getFlairs());
+    } catch (err) {
+      return res.serverError(err);
+    }
```

Comment:
I kept this function since it's nice to have a working api, but this is no longer used, since the flairs are sent as an EJS local when viewing a page anyway. I also refactored this to make it async.